AndroidSDKCore/src/main/java/com/leanplum/Leanplum.java

```diff
@@ -2142,4 +2146,14 @@ private static void parseVariantDebugInfo(JSONObject response) {
       VarCache.setVariantDebugInfo(variantDebugInfo);
     }
   }
-}
+
+  private static void parseSdkCounters(JSONObject response) {
+
+    JSONArray enabledCounters = response.optJSONArray(
+            Constants.Keys.ENABLED_COUNTERS);
+    if (enabledCounters != null) {
+      Set counterSet = new HashSet<>(Arrays.asList(enabledCounters));
+      Log.setEnabledCounters(counterSet);
+    }
+  }
+}
```

Comment:
nit: empty line at end of file is more tidy. You can tell by Github's bias in design showing that it's missing with a red circle and cross.